### DISCUSSION TITLE
Thubnail Size seriously?

### DISCUSSION THREAD

**Discussion by leucome** [spacedriveapp/spacedrive#2508] [Community Support]
No symlink and tiny baby thumbnails.  Is it really designed for creator? 
![Size](https://github.com/spacedriveapp/spacedrive/assets/3796740/64175d8e-262d-4936-ac69-53e541169289)

**Comment by iLynxcat** [comment]
Hey @leucome are you still having this issue on the latest release [Alpha 0.4.0](https://github.com/spacedriveapp/spacedrive/releases/tag/0.4.0)?
